components/js/signal-timing.js

```diff
@@ -67,7 +67,7 @@ var SYSTEM_IDS = {};
 var tau = 2 * Math.PI,
     arc;
 
-var selected_year = "2018";  //  init year selection
+var selected_year = "2019";  //  init year selection
```

Comment:
Is there a way we could solve this that would automatically update so we don't have to do this again next year?

hint:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now